packages/landing/src/components/feature.updates.tsx

```diff
@@ -0,0 +1,140 @@
+import { clsx } from 'clsx';
+import { Component, ReactNode } from 'react';
+import ReactModal from 'react-modal';
+
+interface IFrameConfig {
+  url: string;
+  width: number;
+  height: number;
+  title: string;
+  iframeWrapperClass?: string;
+  iFrameProps?: Partial<JSX.IntrinsicElements['iframe']>;
+}
+
+type FeatureUpdatesProps = {
+  header: string;
+  wrapperClass?: string;
+  id: string;
+  releaseVersion: string;
+  enabled: boolean;
+  children?: ReactNode;
+} & (
+  | { bigImage: string; smallImage: string; iframe?: never }
+  | { bigImage?: never; smallImage?: never; iframe: IFrameConfig }
+);
+
+type FeatureUpdatesState = {
+  showModal: boolean;
+};
+
+/**
+ * FeatureUpdates is a re-implementation of @linzjs/lui -> LuiUpdatesSplashModal module,
+ * This use to enable a one off pop up screen for introduce new feature of the recent release.
+ *
+ */
+export class FeatureUpdates extends Component<FeatureUpdatesProps, FeatureUpdatesState> {
+  constructor(props: FeatureUpdatesProps) {
+    super(props);
+    const currentVersion = window.localStorage.getItem(this.props.id);
+
+    this.state = {
+      showModal: this.props.enabled && this.props.releaseVersion.trim() !== currentVersion,
+    };
+  }
+
+  handleClose = (): void => {
+    this.setState({ showModal: false });
+    window.localStorage.setItem(this.props.id, this.props.releaseVersion);
+  };
+
+  renderFeatureMedia(): ReactNode {
+    const { bigImage, smallImage, iframe } = this.props;
+    if (iframe) {
```

Comment:
we have no plans for the iframe variant, could we rip all this out?

Reply:
All removed